### ISSUE TITLE
rhino3dm 7.14.2 won't install through pip (invalid wheel!)

### ISSUE THREAD

**Issue by fstwn** [mcneel/rhino3dm#464]
Dear all,
we had this bug alredy some weeks ago. `rhino3dm` 7.14.2 won't install on win through pip:
```
(base) > pip install rhino3dm
Collecting rhino3dm
  Downloading rhino3dm-7.14.2-cp37-cp37m-win_amd64.whl (2.0 MB)
     ---------------------------------------- 2.0/2.0 MB 11.5 MB/s eta 0:00:00
ERROR: rhino3dm has an invalid wheel, rhino3dm has an invalid wheel, .dist-info directory not found
```

Thanks for taking a look, it seems to me this is related with the current PyPi release workflow? Maybe something in there needs changing...

**Comment by fstwn** [comment]
Alright, thank you for clarifying! I don't wanted to bother but raise awareness since I noticed while trying to update my research environments this morning :)
Best wishes, Max

**Comment by sbaer** [comment]
Hopefully this is fixed with 7.15 release. We still need to fix our github actions for release.

**Comment by fraguada** [comment]
This should be fixed with the 8.0.0 release
